ClosedXML/Excel/Ranges/XLRangeBase.cs

```diff
@@ -1179,14 +1231,18 @@ private IXLRangeColumns InsertColumnsBeforeInternal(Boolean onlyUsedCells, Int32
                         var styleToUse = Worksheet.Internals.RowsCollection.ContainsKey(ro)
                                              ? Worksheet.Internals.RowsCollection[ro].Style
                                              : Worksheet.Style;
-                        rangeToReturn.Row(ro).Style = styleToUse;
+                        using (var row = rangeToReturn.Row(ro))
+                            row.Style = styleToUse;
                     }
 
                 }
             }
 
             if (nullReturn)
+            {
+                rangeToReturn.Dispose();

```

Comment:
RangeToReturn is returned from the method. Therefore, we must Dispose it only if nullReturn.